assets/scripts/check-microk8s-macos.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+set -e
+
+#===========
+# Parameters
+#===========
+
+PASSWORD=$1
+ASSETS_FOLDER=$2
+
+#================
+# Verify MicroK8s
+#================
+
+CONFIGURE_MICROK8S=false
+if echo "$PASSWORD" | microk8s version >/dev/null; then
+    echo "microk8s is installed"
+else
+    echo "microk8s is not installed"
+
+    echo "$PASSWORD" | sudo snap install microk8s --classic --channel=1.26/stable
+
+    CONFIGURE_MICROK8S=true
+
+    echo "$PASSWORD" | sudo -S usermod -a -G microk8s $USER
+    echo "$PASSWORD" | sudo -S chown -R $USER ~/.kube
+
+    # Remove previous context from config
+    if kubectl config view -o jsonpath='{.contexts}' | grep 'microk8s'; then
+        kubectl config delete-context microk8s
+    fi
+
+    # Remove previous cluster from config
+    if kubectl config view -o jsonpath='{.clusters}' | grep 'microk8s-cluster'; then
+        kubectl config delete-cluster microk8s-cluster
+    fi
+
+    # Remove previous user from config
+    if kubectl config view -o jsonpath='{.users}' | grep 'microk8s-admin'; then
+        kubectl config delete-user microk8s-admin
+    fi
+
+    # Ensure the certificate is accessible. Ref: https://askubuntu.com/a/720000/1558816
+    echo "$PASSWORD" | sudo -S chmod a+rwx /var/snap/microk8s/current/certs/
+    echo "$PASSWORD" | sudo -S chmod a+rwx /var/snap/microk8s/current/certs/ca.crt
+
+    # Ref: https://discuss.kubernetes.io/t/use-kubectl-with-microk8s/5313/6
+    echo kubectl config set clusters.microk8s.certificate-authority-data --server=https://127.0.0.1:16443/
```

Comment:
I guess here we need to create a cluster with name `microk8s` as it will be used in line 51 as --cluster argument.